lib/connection.js

```diff
@@ -268,19 +268,22 @@ Connection.prototype.read = function(data) {
         // last time so now we want to ignore it
         err = new Error(util.format('Memcache returned an error: %s\r\nFor key %s', data, deferred.key));
     } else if (data.match(/^VALUE .+/)) {
-        var spl = data.match(/^VALUE (.+) ([0-9]+) ([0-9]+)$/);
+        var spl = data.match(/^VALUE ([^ ]+) ([0-9]+) ([0-9]+)( ([0-9]+))?$/);
```

Comment:
Just curious, did you mean to include the space in the last selector in this regex?

That will lead to a trailing space in the cas variable won't it?